api/migrations/versions/38dde1052c5e_.py

```diff
@@ -0,0 +1,56 @@
+"""empty message
+
+Revision ID: 38dde1052c5e
+Revises: 351c26992f23
+Create Date: 2019-01-23 19:56:29.948294
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '38dde1052c5e'
+down_revision = '351c26992f23'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import table
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    event = op.create_table('Event',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('Name', sa.String(length=255)),
+    sa.Column('Description', sa.String(length=255)),
+    sa.Column('StartDate', sa.DateTime()),
+    sa.Column('EndDate', sa.DateTime()),
+    sa.PrimaryKeyConstraint('id')
+    )
+
+    op.create_table('Event_Role',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('event_id', sa.Integer()),
+    sa.Column('user_id', sa.Integer()),
+    sa.Column('role', sa.String(length=50)),
+    sa.ForeignKeyConstraint(['user_id'], ['app_user.id'] ),
+    sa.ForeignKeyConstraint(['event_id'], ['Event.id'] ),
+    sa.PrimaryKeyConstraint('id')
+    )
+
+    ### end Alembic commands ###
+    op.bulk_insert(event,
+    [
+        {
+            'id': 1,
+            'name': 'indaba 2019',
+            'description': """ The Deep Learning Indaba 2019, Kenyatta University, Nairobi, Kenya """,
+            'StartDate': '25 August 2019',
+            'EndDate': '31 August 2019'
+        }
+    ])
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('Event')
+    op.drop_table('Event_Role')
```

Comment:
From the error on CircleCI, it looks like the `downgrade()` is failing. The error it's getting is 
> sqlalchemy.exc.InternalError: (psycopg2.InternalError) cannot drop table "Event" because other objects depend on it

It seems that `Event_Role `depends on `Event` so swapping the two lines around should get the test to pass since you'll be dropping the dependent table first. Hope this helps.

Reply:
Yes, this resolved the downgrade issue, thanks!